src/components/ChannelsAvailability/ChannelsAvailabilityWithPermission.tsx

```diff
@@ -0,0 +1,25 @@
+import { hasPermission } from "@saleor/auth/misc";
```

Comment:
@AlicjaSzu an meself figured it'd make more sense to make a general permissions wrapper component for these kind of situations